packages/js/components/src/file-upload-area/README.md

```diff
@@ -0,0 +1,29 @@
+# FileUploadArea
```

Comment:
I planned to use the [FormFileUpload component](https://developer.wordpress.org/block-editor/reference-guides/components/form-file-upload/),  but it renders everything inside a button. Since we needed to add another button inside the button, I leaned towards creating our own component to have a clickable area to upload files.